grpc/src/main/protobuf/transactions_api.proto

```diff
@@ -0,0 +1,34 @@
+syntax = "proto3";
+package com.wavesplatform.api.grpc;
+import "recipient.proto";
+import "transaction.proto";
+import "google/protobuf/empty.proto";
+
+service TransactionsApi {
+    rpc GetTransactionsByAddress (TransactionsByAddressRequest) returns (stream SignedTransaction);
+    rpc GetTransactionById (TransactionByIdRequest) returns (SignedTransaction);
```

Comment:
Both `GetTransaction*` methods should return `SignedTransactionAndHeight`. Also how about
```suggestion
    rpc GetTransactions (TransactionsRequest) returns (SignedTransactionWithHeight);
    rpc GetTransaction (TransactionRequest) returns (SignedTransactionWithHeight);
```
`GetTransactions` also returns transactions for an alias, not only for an address.